gitlab-pipeline/stage/test.yml

```diff
@@ -60,6 +60,10 @@ test:backend-integration:open_source:
     - mv stage-artifacts/mender-artifact-linux integration/backend-tests/downloaded-tools/mender-artifact
     # copy for pre 2.4.x releases
     - cp integration/backend-tests/downloaded-tools/mender-artifact integration/backend-tests/mender-artifact
+    - if [ -n "$MENDER_TEST_CONTAINERS_CANDIDATE_TAG" ]; then
+    -   sed -i.bak -e "s,mendersoftware/mender-test-containers:backend-integration-testing,$MENDER_TEST_CONTAINERS_CANDIDATE_TAG," integration/backend-tests/docker/docker-compose.backend-tests.yml
+    -   diff -u integration/backend-tests/docker/docker-compose.backend-tests.yml.bak integration/backend-tests/docker/docker-compose.backend-tests.yml || true
```

Comment:
You need to log in here too into the GitLab Container Registry. See [this line](https://github.com/mendersoftware/mendertesting/pull/315/files#diff-776b076f35101fbcb0388f4338742d31b49d2bdd37618d0862c77e6006501b0cR104).

Reply:
The lines above are for docker.io and registry.mender.io, still missing the log in for gitlab.com (I know, it is messy...)

Reply:
> The lines above are for docker.io and registry.mender.io, still missing the log in for gitlab.com (I know, it is messy...)

yes sorry I missed that.